### ISSUE TITLE
Replace insert-text-at-cursor library from examples for newer text-field-edit

### ISSUE THREAD

**Issue by rodrigoaguilera** [nolanlawson/emoji-picker-element#478]
Using a library that didn't receive any updates for 5 years can be an issue for some folks. At least it was for me so I went on a search for something more maintained.

Seems like
https://www.npmjs.com/package/text-field-edit

Can be a good replacement. Any concerns?
If the maintainers are OK with this I can prepare a PR.

**Comment by nolanlawson** [comment]
This is just an example for documentation, so I don't have a strong opinion on which dependency to use. In fact it'd probably be best to remove the dependency entirely if that's possible.

**Comment by rodrigoaguilera** [comment]
I agree is just that I wanted the exact same thing as the demo so I want to save others from the search.
I don't think is easy to remove the dependency as the library takes care of many details when performing its focused task.

Please review the PR
https://github.com/nolanlawson/emoji-picker-element/pull/481
